src/main/java/com/synopsys/integration/jenkins/scan/global/ApplicationConstants.java

```diff
@@ -0,0 +1,28 @@
+package com.synopsys.integration.jenkins.scan.global;
+
+public class ApplicationConstants {
+
+    public static final String APPLICATION_NAME = "synopsys-security-scan";
+    public static final String DISPLAY_NAME = "Synopsys Scan";
+    public static final String PIPELINE_NAME = "synopsys_scan";
+    public static final String BRIDGE_ARTIFACTORY_URL = "https://sig-repo.synopsys.com/artifactory/bds-integrations-release/com/synopsys/integration/synopsys-bridge/";
+    public static final String SYNOPSYS_BRIDGE_RUN_COMMAND = "./synopsys-bridge";
+    public static final String SYNOPSYS_BRIDGE_LATEST_VERSION = "latest";
+    public static final String PLATFORM_LINUX = "linux64";
+    public static final String PLATFORM_WINDOWS = "win64";
+    
+    public static final String BLACKDUCK_URL_KEY = "blackduck_url";
+    public static final String BLACKDUCK_API_TOKEN_KEY = "blackduck_api_token";
+    public static final String BLACKDUCK_INSTALL_DIRECTORY_KEY = "blackduck_install_directory";
```

Comment:
We may need to check, if the parameter names should be concatenated with BRIDGE.

Reply:
As @sig-tithi said, we can refactor this later with all other parameters. 

N.B. _synopsys-action_ repo also follows this pattern